script/lib/utils.js

```diff
@@ -82,7 +82,7 @@ async function getCurrentBranch (gitDir) {
       '--remote'
     ], gitDir)).split('\n')
 
-    branch = branches.filter(b => b.trim() === 'master' || RELEASE_BRANCH_PATTERN.test(b.trim()))[0]
+    branch = branches.filter(b => b.trim() === 'master' || b.trim() === 'origin/master' || RELEASE_BRANCH_PATTERN.test(b.trim()))[0]
```

Comment:
this should probably be a little more general for forks: e.g `upstream/master` should work as well

Reply:
We don't handle `upstream/` as a prefix below, is there a case in sudowoodo where the remote could be called `upstream`?

Reply:
verdict: follow-up concern :)